### DISCUSSION TITLE
How to change the direction of rotation and blades initial position.

### DISCUSSION THREAD

**Discussion by clbanjai** [OpenFAST/openfast#2335] [Q&A] [answered]
Hi all,

I was wondering if anyone knew if there was a way of inverting the rotational direction of the blade turbines. I also wanted to change the initial position of the blades, let's say instead of having blade starting at 0 degrees above the horizontal axis it would start at 30 degrees above the horizontal, I was thinking of maybe changing the initial azimuth degree but I am not fully sure if that is correct.

Thanks for the help,

**Comment by jjonkman** [comment, marked as answer]
Dear @clbanjai,

OpenFAST does not currently support the modeling of wind turbine rotors that spin counter clockwise when looking downwind, as already reported in the following OpenFAST issue: https://github.com/OpenFAST/openfast/issues/1194.

To set the initial rotor orientation, I agree that you change the initial azimuth angle in ElastoDyn (input parameter `Azimuth`).

Best regards,
